.buildkite/lint.rayci.yml

```diff
@@ -88,7 +88,7 @@ steps:
 
   - label: ":lint-roller: lint: linkcheck"
     instance_type: medium
-    command: make -C doc/ linkcheck
+    commands:
+      - if [[${BUILDKITE_BRANCH} == 'master']]; then make -C doc/ linkcheck_all; else make -C doc/ linkcheck; fi
```

Comment:
i think a space is require after the `[[` and before `]]`

`[[ ${BUILDKITE_BRANCH} == 'master' ]]`